utils/results/conversion/simplejsonparser/simplejsonparser.go

```diff
@@ -342,6 +343,12 @@ func PrepareSimpleJsonVulnerabilities(target results.ScanTarget, scaResponse ser
 	return vulnerabilitiesRows, err
 }
 
+// CheckIfMalicious determines if a summary indicates a malicious vulnerability/violation
+func CheckIfMalicious(summary string) bool {
+	summary = strings.ToLower(summary)
+	return strings.HasPrefix(summary, "malicious") || strings.HasPrefix(summary, "the docker image malicious")
+}
```

Comment:
I don't think it's a good idea to parse and infer if malicious or not.
maybe the server can add an attribute in response that specify this.